### ISSUE TITLE
Remove or import to '3rdparty' any private repos in package.json

### ISSUE THREAD

**Issue by brendanarnold** [DFE-Digital/polis-whitelabel#27]
`package.json` refers to some private repositories - generally for legacy projects that shoul dbe refactored out soon. 

If not required we should remove the references else import the compiled javascript into `js/3rdparty` so that the code is stable.

**Comment by patcon** [comment]
Alternative approach is to _not_ pull it into local repo, but work to upstream the changed, ornmigrate away. My sense is that forking the repos and keeping them separate is cleaner (so as to not version-controll a slightly changed package in with our repo, whilst detaching from version history). But your call! Happy to discuss.

(I'd prefer to deprecate 3rdparty, not expand it)

**Comment by brendanarnold** [comment]
Current problematic dependencies ...

```
// client-participation/package.json
...
    "backbone": "https://github.com/sirodoht/compdemocracy-backbone",
    "handlebars-v1": "https://github.com/sirodoht/compdemocracy-handlebars-v1", // Can definitely remove this now
    "handlebones": "https://github.com/sirodoht/compdemocracy-handlebones",
...
```

**Comment by brendanarnold** [comment]
@patcon Yes agree the ideal is to migrate away - I see moving to `3rdparty` as a tactical fix until we move to React or similar more modern front-end.
